src/modules/LanItems.rb

```diff
@@ -2203,17 +2203,24 @@ def enableCurrentEditButton
 
     # Creates eth emulation for s390 devices
     #
-    # @param [Hash] an s390 device description as obtained from AY profile
-    def createS390Device(rule)
+    # @param [Hash] an s390 device description (e.g. as obtained from AY profile).
+    # if not present then initialization of S390 specific LanItems' variables is
+    # expected to be done out of this function.
+    def createS390Device(dev_attrs = {})
       Builtins.y2milestone("creating device s390 network device, #{rule}")
```

Comment:
BTW I can make ruby-lint find this bug, as well as the one this PR is fixing!
But they are hidden among 688 false positives.